src/Resources/js/modules/custom-functions.js

```diff
@@ -40,7 +40,12 @@ Mealz.prototype.copyToClipboard = function() {
             } else {
                 guestMenuLinkInput.trigger('select');
             }
-            navigator.clipboard.writeText(result);
+            // Clipboard copy only works in secure context, redirect http -> https
+            if(window.isSecureContext) {
```

Comment:
this is not supported in normal internet explorers
https://developer.mozilla.org/en-US/docs/Web/API/isSecureContext#browser_compatibility

alternative you can check with
`
window.location.protocol == 'https:'
`

Reply:
Done.